sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/util/IdentityConstants.java

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.identity.implementation.util;
+
+public class IdentityConstants {
+    /**
+     * The Single Sign On ID.
+     */
+    public static final String DEVELOPER_SINGLE_SIGN_ON_ID = "04b07795-8ddb-461a-bbee-02f9e1bf7b46";
```

Comment:
Single sign on ID? This is the client ID of Azure CLI.

Reply:
Yeah the CLI ID is being used temporarily here in other languages too, until we get other Client Id.